oauth/authentication-oauth2.0.md

```diff
@@ -88,36 +95,45 @@ GET https://app.solvexia.com/authorize?client_id=DDF-AAFBD447-55432-475B-83DB-B5
 | ------------- |------------- | -------------|
 |client_id|`string`|REQUIRED. Client id that you received when you created an application in SolveXia.|
 |response_type|`string`|REQUIRED. will always be "code".|
-|redirect_uri|`string`|OPTIONAL. If not supplied it will use configured redirect url of the oauth application, if supplied it must match the oauth application redirect url|
-
-##### Response
+|redirect_uri|`string`|OPTIONAL. If not supplied it will use configured redirect url of the oauth application, if supplied it must match the oauth application redirect url.|
+|state|`string`|OPTIONAL. Specifies any string value that your application uses to maintain state between your authorization request and the authorization server's response.|
 
-Will redirect to login page
+After SolveXia authorization server successfully verifies your parameters it will redirect user to the login page and further authorization form.
```

Comment:
I think this makes more sense: "After the SolveXia authorization server successfully verifies your parameters, it will redirect the user to the login page and other authorization forms."

Reply:
Thanks, fixed.